.github/workflows/esp32-mkimage.yaml

```diff
@@ -125,3 +137,22 @@ jobs:
         name: atomvm-${{ matrix.soc }}-image
         path: ./src/platforms/esp32/build/atomvm-${{ matrix.soc }}-*.img
         if-no-files-found: error
+
+    - name: "Rename and write sha256sum"
+      if: startsWith(github.ref, 'refs/tags/')
+      shell: bash
+      working-directory: src/platforms/esp32/build
+      run: |
+        ATOMVM_IMG="AtomVM-${{ matrix.soc }}-${{ github.ref_name }}.img"
```

Comment:
Why upcase?  (I prefer downcase `atomvm`, but maybe that’s just me)

Reply:
Just for consistency reasons, as we do with unix platform output.

Reply:
Okay, I mean, I can live with that, especially if this the "external" name.